app/models/user_action_event.rb

```diff
@@ -3,7 +3,9 @@
 class UserActionEvent < ApplicationRecord
   has_many :user_actions, dependent: :restrict_with_exception
 
+  EVENT_TYPES = %w[authentication].freeze
```

Comment:
Why do we have event types defined here? It has to be defined in the yml anyways so it doesn't seem like there's an advantage for the class to validate the values